content/en/platform/corda/5.0-dev-preview-2/developing/api/amqp-serialization-default-evolution.md

```diff
@@ -0,0 +1,287 @@
+---
+date: '2022-09-21T14:27:00+01:00'
+title: "Default Class Evolution"
+menu:
+  corda-5-dev-preview:
+    identifier: corda-5-dev-preview-api-serialisation-default-evolution
+    parent: corda-5-dev-preview-api-serialisation
+    weight: 8000
+section_menu: corda-5-dev-preview
+---
+
+
+
+# Default Class Evolution
+
+
+Whilst more complex evolutionary modifications to classes require annotating, Corda’s serialization
```

Comment:
This is hard to follow..
Maybe:

Whilst more complex evolutionary modifications to classes require annotating, Corda’s serialization
framework supports several minor modifications to classes without any external modification to
the actual code changes.

Reply:
So the suggestion is to replace 'save' with 'to'. My understanding of the sentence is that save means 'other than' or 'except'. I think using the word 'to' changes the meaning but the sentence is still true.

Would you prefer 'to' or 'except'?

Reply:
"except" please. I had written it with that initially but was unsure if I'd missed understood! Thanks

Reply:
Done